compiler/src/iree/compiler/Dialect/Flow/Transforms/InjectDispatchTracing.cpp

```diff
@@ -42,14 +70,22 @@ struct InjectDispatchTracingPass
       builder.create<IREE::Flow::TensorTraceOp>(
           dispatchOp.getLoc(),
           builder.getStringAttr(entryPointName + " inputs"),
```

Comment:
Taking the second look, I think users could be confused with it. Because it does not print the actual inputs/outputs. Maybe we should add such information to the string. Perhaps we can collect the set of encoded operands and attach them in the name like " inputs with decoded/row-major [1, 2, 3, 5, 8] operands".